### PR TITLE
Update for DnD5e 3.3.x

### PR DESCRIPTION
**Pull request by Szerina92**
Update for the new version.
I'm testing/using it in V12 for some time now and I haven't noticed any problems.
Hope it helps!

### CODE REVIEW

**Comment by Szerina92** [comment]
@mhilbrunner forgot to ping :3

File: languages/de.json [hunk 1]
```diff
@@ -1290,6 +1362,7 @@
 "DND5E.SavePromptTitle": "{ability} Rettungswurf",
 "DND5E.SavingThrow": "Rettungswurf",
 "DND5E.SavingThrowDC": "SG {dc} {ability}srettungswurf",
+"DND5E.SavingThrowRoll": "Werfe {ability}srettungswurf",
```
Comment (mhilbrunner) [inline languages/de.json:1365]: Oh, if we just had more powerful translation features. "Intelligenz**s**rettungswurf" unfortunately is wrong. We could do "{ability}-Rettungswurf" for "Intelligenz-Rettungswurf", maybe. Anyway, I note the existing translation in the line above already does this also, so we can fix this later. No worries for now.

File: languages/de.json [hunk 2]
```diff
@@ -1360,6 +1434,7 @@
 "DND5E.SkillBonuses": "Fertigkeitsboni",
 "DND5E.SkillConfigurationHint": "Übung in Fertigkeit und Boni bearbeiten.",
 "DND5E.SkillConfigure": "Fertigkeit bearbeiten",
+"DND5E.SkillsConfig": "Fertigkeit bearbeiten",
```
Comment (mhilbrunner) [inline languages/de.json:1437]: Considering this, in contrast to the preceding one, is plural, it maybe should be "Fertigkeiten"? Not sure, would need to check in context.

**Review by mhilbrunner — APPROVED** [review]

**Comment by Szerina92** [comment]
image
Right, should be the plural "Fertigkeiten", my bad.
It's from the new NPC sheet.

**Comment by mhilbrunner** [comment]
@Szerina92 Thanks for the ping! Released version [3.3.0](https://github.com/mhilbrunner/foundryvtt-dnd5e-lang-de/releases/tag/3.3.0) with these changes, and marked the module V12-ready. Thanks for your efforts!

Feel free to follow up on those other things in new PRs if you want, but nothing too critical here :)

**Pull request merged**